static/src/javascripts/projects/common/modules/atoms/youtube.js

```diff
@@ -288,7 +288,6 @@ const getIFrameBehaviour = (
         autoplay:
             ((isVideoArticle && isInternalReferrer && isMainVideo) ||
                 isFront) &&
-            !isAutoplayBlockingPlatform &&
```

Comment:
yep, good point. This was removed becuase we were going to autoplay on all breakpoints. Because this isn't happening now, I've added a check so we only autoplay on mobile if its a vertical video